multiversx_sdk/network_providers/user_agent.py

```diff
@@ -9,7 +9,7 @@
 
 def extend_user_agent(user_agent_prefix: str, config: NetworkProviderConfig):
     if config.client_name is None:
-        logger.warning("Please provide the client name of the application that uses the SDK. It will be used for metrics.")
+        logger.warning("We recommend providing the `client_name` when instantiating a NetworkProvider (e.g. ProxyNetworkProvider, ApiNetworkProvider). This information will be used for metrics collection and improving our services.")
```

Comment:
Also, not `warning`, but `info` instead.

Reply:
Done